conf/wustl_htcf.config

```diff
@@ -0,0 +1,40 @@
+// Forked from https://github.com/nf-core/configs/blob/master/conf/prince.config
+
+def singularityDir = set_singularity_path()
+
+params {
+  config_profile_description  = """
+                                  WUSTL High Throughput Computing Facility cluster profile provided by nf-core/configs.
+                                  Run from your scratch directory, the output files may be large!
+                                  Please consider running the pipeline on a compute node the first time, as it will be pulling the docker image, which will be converted into a singularity image, which is heavy on the login node. Subsequent runs can be done on the login node, as the docker image will only be pulled and converted once. By default, the images will be stored in $singularityDir
+                                """.stripIndent()
+  config_profile_contact      = "Gavin John <gavinjohn@wustl.edu>"
+  config_profile_url          = "https://github.com/nf-core/configs/blob/master/docs/wustl_htcf.md"
+}
+
+spack {
+  enabled = true
+}
+
+singularity {
+  enabled  = true
+  cacheDir = singularityDir
+}
+
+process {
+  beforeScript = "exec \$( spack load --sh singularity )"
+  executor     = "slurm"
+}
+
+def set_singularity_path() {
+  def lab = System.getenv("LAB")
+  if (lab == null){
+    System.err.println("")
+    System.err.println((String)((char)27) + "[33m" + "WARNING: wustl_htcf.config requires the LAB environment variable to be set. Please add the following line to your .bashrc:" + (String)((char)27) + "[30m")
+    System.err.println("""export LAB=\$(id -nG "\$(whoami)" 2> /dev/null | awk 'NF>1{print \$NF}')""")
+    System.err.println((String)((char)27) + "[0m")
```

Comment:
Just double checking - there's no way to actually get the output of `id -nG "$(whoami)" 2> /dev/null | awk 'NF>1{print $NF}'` without an environment variable, right?

Reply:
To be honest you're going very off-standard here with custom Groovy :sweat_smile: , so I don't have experience in this (Sorry). @maxulysse might know?